great-idea-website/index.html

```diff
@@ -15,49 +15,87 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
+    <header>
+    <div class="container">
+      <nav>
+    <a href="#"> Services </a>
+    <a href="#">Product</a>
+    <a href="#"> Vision</a>
+    <a href="#">Features</a>
+    <a href="#">About</a>
+    <a href="#">Contact</a>
+        </nav>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </div>
+</header>
+
+<section class=all-cta>
+  <div class = ctaContent>
+    <h1>
+      Innovation</br>
+      On</br>
+      Demand</br>
+</h1>
+    <button>Get Started</button>
+  </div>
+  <img class = cta-img src="img/header-img.png" alt="Image of a code snippet."/>
+
+</section>
+
+<hr></hr>
+
+<section>
+ <div class = features>
+  <span>Features</span>
+  <p>
+    Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  </p>
+ </div>
+
+<div class = about>
+  <span>About</span>
+<p>
   Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+</p>
+</div>
+</section>
 
 
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
+<div>
```

Comment:
This outer div wrapping the img is not necessary.  A img itself it an html element and can be targeted using regular methods as you are already doing.